peachjam/views/generic_views.py

```diff
@@ -61,6 +75,25 @@ class DocumentListView(ListView):
     # when grouping by date, group by year, or month and year? ("year" and "month-year" are the only options)
     group_by_date = "year"
 
+    def page_title(self):
+        return _("Documents")
+
+    def get_paginator(
+        self, queryset, per_page, orphans=0, allow_empty_first_page=True, **kwargs
+    ):
+        return self.paginator_class(
+            queryset,
+            per_page,
+            orphans=orphans,
+            allow_empty_first_page=allow_empty_first_page,
+            cache_key_prefix=self.cache_key_prefix,
+            **kwargs,
+        )
+
+    @cached_property
```

Comment:
No need to cache this, a regular method is fine.

Reply:
ok, changed